Automattic/jetpack
# VideoPress: Add transform control from video block to videopress/video block

Fixes #28637

## Proposed changes:

* Temporarily adds a transform control to v5 video blocks containing VideoPress-converted videos to v6 to encourage update

### Other information:


## Jetpack product discussion


p1675244289505449/1675182827.670519-slack-C03TA48NSUX

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Before connecting Jetpack, go to the block editor, add a `video` block and upload a video. Save the post.
* Connect Jetpack.
* Add two `video` block, uploading two videos to be processed by VideoPress.
* Add a v6 `videopress/video` block with another video.
* Save the post.
* Check that only the `video` blocks with a VideoPress-converted video display the transform control.
* In the transform control of one of the `video` blocks, click on `Transform blocks to VideoPress`.
* Check that the two `video` blocks processed by VideoPress were transformed to v6 `videopress/video` blocks without loss of data.

https://user-images.githubusercontent.com/8486249/216062108-076b1f1f-1a4a-447f-bb54-fd167bda1e0d.mp4
